k8s/base/prosody-deployment.yml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: prosody
+  name: prosody
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prosody
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: prosody
+    spec:
+      containers:
+      - image: jitsi/prosody:stable-5963
+        name: prosody
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          httpGet:
+            path: /http-bind
```

Comment:
Does this path is available is you enable only websocket and not bosh ?

Reply:
Good question! I just checked:
In the configuration file provided with the `jitsi/prosody` docker image, bosh is always enabled, so this endpoint is always available.

See https://github.com/jitsi/docker-jitsi-meet/blob/master/prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua#L110-L112